packages/react-native-reanimated/src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -584,6 +584,7 @@ export function createAnimatedComponent(
         <Component
           nativeID={nativeID}
           {...filteredProps}
+          jestInlineStyle={this.props.style}
```

Comment:
Do we want to attach it when we're not in Jest environment?

Reply:
Also what about non-inline jest style? It would be more maintainable if it was used here as well.

Reply:
Probably we won't, changed in [e147d2b](https://github.com/software-mansion/react-native-reanimated/pull/6400/commits/e147d2b0eb362ac0a51663dbe9dce8b791a027f5), along with adding prop for non-inline styles